lib/sycamore/sycamore/transforms/merge_elements.py

```diff
@@ -135,7 +135,151 @@ def merge(self, elt1: Element, elt2: Element) -> Element:
 
         return new_elt
 
+class GreedySectionMerger(ElementMerger):
+    def __init__(self, tokenizer: Tokenizer, max_tokens: int, merge_across_pages: bool = True):
+        self.tokenizer = tokenizer
+        self.max_tokens = max_tokens
+        self.merge_across_pages = merge_across_pages
+
+    def preprocess_element(self, element: Element) -> Element:
+        if element.type == 'Image' and "summary" in element.properties:
+            element.data["token_count"] = len(self.tokenizer.tokenize(element.properties['summary'] or ""))
+        else:
+            element.data["token_count"] = len(self.tokenizer.tokenize(element.text_representation or ""))
+        return element
+
+    def postprocess_element(self, element: Element) -> Element:
+        del element.data["token_count"]
```

Comment:
I know we haven't previously but I wonder if keeping token_count around in elements is actually useful, it'll help us to cost estimation for running LLMs across a set of elements. Thoughts? @bsowell too